### PR TITLE
added android release blog post

### PR DESCRIPTION
**Pull request by stevengill**
Blog post for android@7.1.0 release. Please leave feedback

### CODE REVIEW

File: www/_posts/2018-02-23-cordova-android-7.1.0.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    cordova platform add android@7.1.0
+
+To add it explicitly:
+
+    cordova platform add android@7.1.0
+
+<!--more-->
+## Curated Changelog
+* [CB-13879](https://issues.apache.org/jira/browse/CB-13879) updated gradle tools dependency to 3.0.1 for project template
+* [CB-13831](https://issues.apache.org/jira/browse/CB-13831) Update `android-versions` to 1.3.0 to support SDK 27.
```
Comment (dpogue) [inline www/_posts/2018-02-23-cordova-android-7.1.0.md]: I know this is all auto-generated from commit messages, but for the sake of the blog post can we add what Android version SDK 27 corresponds to? I'm guessing that's 8.1?

File: www/_posts/2018-02-23-cordova-android-7.1.0.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+* [CB-13800](https://issues.apache.org/jira/browse/CB-13800) Drop pre-KitKat specific code
```
Comment (dpogue) [inline www/_posts/2018-02-23-cordova-android-7.1.0.md]: `pre-KitKat` -> `pre-4.4 (KitKat)` for clarity? We (and Google for that matter) are very inconsistent with referring to versions by SDK, version number, and name.

File: www/_posts/2018-02-23-cordova-android-7.1.0.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+* [CB-13724](https://issues.apache.org/jira/browse/CB-13724) Updated the **Android** Tooling required for the latest version on both the test project, and the template
+* [CB-13724](https://issues.apache.org/jira/browse/CB-13724) Bump Target SDK to API 27
+* [CB-13646](https://issues.apache.org/jira/browse/CB-13646) Using the deprecated `NDK` by default breaks the build.  Crosswalk users need to specify the Gradle parameters to keep it working.
+* [CB-12218](https://issues.apache.org/jira/browse/CB-12218) Fix consistency of null result message
+* [CB-13571](https://issues.apache.org/jira/browse/CB-13571) Prevent crash with unrecognized **Android** version
+* [CB-13721](https://issues.apache.org/jira/browse/CB-13721) Fix build apps that use `cdvHelpers.getConfigPreference`
+* [CB-13621](https://issues.apache.org/jira/browse/CB-13621) Wrote similar warning to [CB-12948](https://issues.apache.org/jira/browse/CB-12948) on **iOS**. We no longer support `cordova update` command.
```
Comment (dpogue) [inline www/_posts/2018-02-23-cordova-android-7.1.0.md]: I'd trim the first sentence of the commit message off of here. Just "We no longer support `cordova update`" is fine, the mention of iOS is confusing.

File: www/_posts/2018-02-23-cordova-android-7.1.0.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+title:  "Cordova Android 7.1.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Android 7.1.0` has been released! This release fixes various bugs releated to the previous `cordova-android@7.0.0` release.
+
+To upgrade:
+
+    npm install -g cordova
```
Comment (janpio) [inline www/_posts/2018-02-23-cordova-android-7.1.0.md]: I know this is jsut copy/paste from an earlier blog post, but why are this and the next line here? Installing CLI seems out of place here and folder name is pure speculation.

Comment (stevengill) [reply www/_posts/2018-02-23-cordova-android-7.1.0.md]: It is just the default we have used for a long time. Haven't heard any complaints about it before

**Pull request merged**